client/src/pages/Map.js

```diff
@@ -0,0 +1,124 @@
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import MarkerClusterGroup from "react-leaflet-markercluster";
+import CheckInText from "../components/CheckInText";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+export default function Map() {
+  const [map, setMap] = useState(null);
+  const [playGroundData, setPlayGroundData] = useState([]);
+  const [state, setState] = useState();
```

Comment:
Could `state` be more specific? Just asking. What kind of information holds `state` ?